src/SFA.DAS.FAA.Web.UnitTests/Controllers/Apply/WhenPostingApplicationSubmitted.cs

```diff
@@ -1,21 +1,43 @@
-using AutoFixture.NUnit3;
-using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using NUnit.Framework;
+using SFA.DAS.FAA.Application.Queries.Apply.GetApplicationSubmitted;
+using SFA.DAS.FAA.Web.AppStart;
 using SFA.DAS.FAA.Web.Controllers;
 using SFA.DAS.FAA.Web.Infrastructure;
 using SFA.DAS.FAA.Web.Models.Apply;
-using SFA.DAS.Testing.AutoFixture;
+using System.Security.Claims;
 
 namespace SFA.DAS.FAA.Web.UnitTests.Controllers.Apply;
 public class WhenPostingApplicationSubmitted
 {
     [Test, MoqAutoData]
     public async Task And_User_Selected_Yes_Then_Redirect_To_Equality_Flow(
         ApplicationSubmittedViewModel model,
+        string bannerMessage,
+        string govIdentifier,
+        Guid candidateId,
+        Guid applicationId,
+        GetApplicationSubmittedQueryResponse response,
+        [Frozen] Mock<IMediator> mediator,
+        [Frozen] Mock<ICacheStorageService> cacheStorageService,
         [Greedy] ApplyController controller)
     {
+        mediator.Setup(x => x.Send(It.Is<GetApplicationSubmittedQuery>(c =>
+                c.CandidateId.Equals(candidateId)
+                && c.ApplicationId.Equals(applicationId)), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(response);
+        var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
+        {
+            new(CustomClaims.CandidateId, candidateId.ToString()),
+        }));
+        controller.ControllerContext = new ControllerContext
+        {
+            HttpContext = new DefaultHttpContext { User = user }
+        };
         model.AnswerEqualityQuestions = true;
+        cacheStorageService.Setup(x => x.Get<string>($"{govIdentifier}-ApplicationSubmitted"))
```

Comment:
I think this can be removed as well

Reply:
Removed the extra code from the Unit Test